install-dev/upgrade/sql/1.7.5.0.sql

```diff
@@ -4,3 +4,6 @@ SET NAMES 'utf8';
 /* PHP:add_supplier_manufacturer_routes(); */;
 
 /* PHP:ps_1750_update_module_tabs(); */;
+
+ALTER TABLE `PREFIX_product` CHANGE `location` `location` VARCHAR(255) NULL DEFAULT NULL;
+ALTER TABLE `PREFIX_product_attribute` CHANGE `location` `location` VARCHAR(255) NULL DEFAULT NULL;
```

Comment:
I think you could have used MODIFY in order to avoid having to type the column name twice: https://dev.mysql.com/doc/refman/5.5/en/alter-table.html#alter-table-redefine-column